src/runtime/mapArgs.ts

```
@@ -53,7 +53,7 @@ export async function renderExplicit(
     }
 
     const render = engineRenderer(config.__type__, options);
-    return render(config.__value__, ctxt);
+    return render?.(config.__value__, ctxt);
```

It would be better to throw a error here if render is not defined. In practice, for render to come back undefined, something must have gone wrong in the Page Editor or YAML to JSON conversion